tests/core/test_internal.py

```diff
@@ -0,0 +1,50 @@
+import asyncio
+from typing import (
+    Any,
+    Tuple,
+)
+
+import pytest
+
+from lahja import (
+    BaseEvent,
+    BroadcastConfig,
+    Endpoint,
+)
+
+
+class DummyResponse(BaseEvent):
+    property_of_dummy_response = None
+
+    def __init__(self, something: Any) -> None:
+        pass
+
+
+@pytest.mark.asyncio
+async def test_internal_propagation(pair_of_endpoints: Tuple[Endpoint, Endpoint]) -> None:
+
+    endpoint1, endpoint2 = pair_of_endpoints
+
+    async def broadcast_dummies() -> None:
+        while True:
+            # We are broadcasting internally on endpoint1
+            endpoint1.broadcast(
+                DummyResponse("Dummy"),
+                BroadcastConfig(internal=True)
```

Comment:
not for this PR.

It might be nice to be able to shortcut this as `endpoint.broadcast(event, internal=True)`